news/12308.removal.rst

```diff
@@ -0,0 +1 @@
+Remove support for discovering installed ``egg`` distributions, on Python 3.11+.
```

Comment:
This is not correct, as we still discover egg distributions that are listed to easy-install.pth.  Not sure how I'm gonna explain this :sweat: